torch/_inductor/triton_bundler.py

```diff
@@ -105,6 +109,14 @@ def begin_compile(cls) -> None:
         assert cls._entries is None
         cls._entries = []
 
+    @classmethod
+    def end_compile(cls) -> None:
+        """
+        Finalizes the TritonBundler. If collect is not yet called, it
+        discards the current bundle.
+        """
+        cls._entries = None
```

Comment:
This is not blocking this PR since you're in a rush, but this is something where it's suuuuper useful to have debug() log saying that end_compile was called.